server/routes/api/choropleth.py

```diff
@@ -210,3 +211,53 @@ def coerce_geojson_to_righthand_rule(geoJsonCords, obj_type):
         return geoJsonCords
     else:
         assert False, f"Type {obj_type} unknown!"
+
+# Defines the map from higher geos to their respective subgeos.
+SUB_GEO_LEVEL_MAP = {
+    "Country": "AdministrativeArea1",
+    "AdministrativeArea1": "AdministrativeArea2",
+    "AdministrativeArea2": "City"
+}
+@bp.route('child/statvars')
+def child_statvars():
+    """
+    Gets all statistical variables available for a particular sublevel of a
+        dcid.
+    API Params:
+        dcid: The place dcid to pull information for, as a string.
+        level: The level of children to pull for, as a string. If omitted,
+            then the subgeo is computed.
+    Example Query:
+        api/place/child/statvars?dcid=country/USA&level=State
+        Returns all statistical variables that are value for states of the USA.
+    Returns:
+        A json list of all the available statistical variables.
+    """
+    # Get required params.
+    dcid = flask.request.args.get("dcid")
+    if not dcid:
+        return flask.jsonify({"error": "Must provide a 'dcid' field!"}, 400)
+    requested_level = get_sublevel(dcid,
+                                   flask.request.args.get("level"))
+
+    # Get sublevels.
+    print(dcid)
```

Comment:
Remove this and other debugging statements

Reply:
Removed all